src/Storefront/Resources/views/storefront/block/cms-block-cross-selling.html.twig

```diff
@@ -1,11 +1,9 @@
 {% block block_cross_selling %}
     {% set element = block.slots.getSlot('content') %}
     {% set columns = 1 %}
-    {% if element.data.crossSellings.elements is defined and element.data.crossSellings.elements|filter(item => item.total > 0)|length > 0 %}
```

Comment:
I guess the is defined condition can stay here?

Reply:
If you keep the code like this:
`{% if element.data.crossSellings.elements is defined %}`

There is still nothing be rendered in the frontend because the element within the block changes. If you drag and drop a cross selling block in the shopping experiences and then change the element to a text element for example the element.data is different. In the other elements there is no "crossSellings"-Array field.

I tested it by changing the element within the cross-selling block with a text element. Take a look:
![image](https://user-images.githubusercontent.com/40269795/178726219-d6eb0915-58a7-46c2-96d8-4bce1da46294.png)

Nothing is being rendered because of the if-condition in the cross-selling block. Now if you just remove the if statement the block is being rendered because this code gets executed now: 
```
<div class="col-12" data-cms-element-id="{{ element.id }}">
            {% block block_cross_selling_inner %}
                {% sw_include "@Storefront/storefront/element/cms-element-" ~ element.type ~ ".html.twig" ignore missing %}
            {% endblock %}
        </div>
```

Now we see the text in the frontend because the frontend code for the selected element gets rendered, even if its not a cross selling element:
![image](https://user-images.githubusercontent.com/40269795/178726608-af3bef61-2dcf-4611-ab30-adddfd4effe1.png)